inc/ticketrecurrent.class.php

```diff
@@ -371,17 +371,17 @@ function computeNextCreationDate($begin_date, $end_date, $periodicity, $create_b
          return 'NULL';
       }
 
-      // First occurence of creation
       $calendar = new Calendar();
       if ($calendars_id && $calendar->getFromDB($calendars_id) && $calendar->hasAWorkingDay()) {
          // Base computation on calendar if calendar is defined
+
          $occurence_date = $calendar->computeEndDate(
             $begin_date,
-            1, // 1 second delay for first occurence as calendar method does not handle 0 delay
+            0, // 0 second delay to get the first working "second"
             0,
-            $periodicity_in_seconds >= DAY_TIMESTAMP
+            false
```

Comment:
Computation of first available time slot was not working as expected in `$work_in_days` mode, as it was moving to next available day when tests were launched before first working hour of the day.